showcase/tests/acceptance/components/hds/app-side-nav-test.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { visit } from '@ember/test-helpers';
+import { setupApplicationTest } from 'showcase/tests/helpers';
+import { a11yAudit } from 'ember-a11y-testing/test-support';
+
+module('Acceptance | Component | hds/app-side-nav', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('Components/app-side-nav page passes a11y automated checks', async function (assert) {
+    let axeOptions = {
+      rules: {
+        listitem: {
+          enabled: false,
```

Comment:
Question: is there a selector you could use to not disable this on the whole page?

Reply:
There may be but let's look at improvements to this as a follow on. This is all taken from the existing SideNav.

Reply:
I will add this to the list of things to look at in a follow on. (Will investigate more at that time if there's any improvement we could do that would make sense.)

Reply:
We haven't really nailed the axe config to narrow down exceptions. We can track this in a separate task